index.html

```diff
@@ -96,51 +97,51 @@ <h1 class="brand-heading">Planktos</h1>
+                Planktos uses <a target="_blank" href="https://webtorrent.io">WebTorrent</a> to convert users of your website into seeders, enabling them to serve static content to other users. This reduces server bandwidth and makes your website more resilient.
             </div>
         </div>
         <div class="row" style="padding-bottom: 45px">
             <div class="col-md-4 col-md-offset-2">
                 <h4>
-                    <i class="fa fa-bolt" aria-hidden="true"></i>
-                    Fast &amp; Scalable
+                    <i class="fa fa-code" aria-hidden="true"></i>
+                    Simple &amp; Easy Integration
                 </h4>
             </div>
             <div class="col-md-4 text-justify">
-              With more users, downloads get faster.
+                Using Planktos is as simple as including the Planktos install script and using the Planktos command line interface to bundle your website's static files into a torrent.
             </div>
         </div>
         <div class="row" style="padding-bottom: 45px">
             <div class="col-md-4 col-md-offset-2">
                 <h4>
-                    <i class="fa fa-lock" aria-hidden="true"></i>
-                    Secure
+                    <i class="fa fa-life-ring" aria-hidden="true"></i>
+                    Avoid downtime
                 </h4>
             </div>
-            <div class="col-md-4 text-justify">
-              All peer downloaded files are verified with sha1 as required by the bittorrent spec.
+            <div class="col-lg-4 text-justify">
+                Planktos can serve your website even if your web servers go down. Any users that visited your website prior to an outage will still be able to access it.
             </div>
         </div>
         <div class="row" style="padding-bottom: 45px">
             <div class="col-md-4 col-md-offset-2">
                 <h4>
-                    <i class="fa fa-code" aria-hidden="true"></i>
-                    Simple &amp; Easy Integration
+                    <i class="fa fa-bolt" aria-hidden="true"></i>
+                    Customizable
                 </h4>
             </div>
             <div class="col-md-4 text-justify">
-              The Planktos service worker intercepts HTTP GET requests and retreives the files over bittorrent. No code changes required other than the one liner to register the service worker.
+                Planktos has sensible defaults and a simple interface for customization, allowing you to tweak it to your needs.
             </div>
         </div>
         <div class="row" style="padding-bottom: 45px">
             <div class="col-md-4 col-md-offset-2">
                 <h4>
-                    <i class="fa fa-files-o" aria-hidden="true"></i>
-                    Supports all files
+                    <i class="fa fa-bolt" aria-hidden="true"></i>
+                    Fast &amp; Scalable
```

Comment:
I think this is one of the core benefits of planktos. Maybe it should be higher up?